go/src/socialapi/workers/payment/api/card_test.go

```diff
@@ -0,0 +1,166 @@
+package api
+
+import (
+	"encoding/json"
+	"fmt"
+	"koding/db/mongodb/modelhelper"
+	"socialapi/models"
+	"socialapi/rest"
+	"socialapi/workers/common/tests"
+	"testing"
```

Comment:
import order - sometimes `goimport` gets it right and format correctly: https://github.com/koding/koding/pull/8882/files#diff-2be48698b8c33592e1f90e3fa58e88cfR3